### PR TITLE
removed openmm deprecation warning

### PR DESCRIPTION
**Pull request by phuddha**
Removed smtk.openmm deprecation warnings.

### CODE REVIEW

File: python-libraries/nanover-openmm/src/nanover/openmm/converter.py [hunk 1]
```diff
@@ -9,11 +9,11 @@
     # Mypy does not find State in the C module; we ignore the error
     from openmm import State  # type: ignore[attr-defined]
 except (ImportError, ModuleNotFoundError):
-    from simtk.openmm import State
+    from openmm import State
```
Comment (jbarnoud) [inline python-libraries/nanover-openmm/src/nanover/openmm/converter.py:12]: Here, and just below, we do not need the try/except construct anymore. The code used to say, "Try importing `State` from `openmm`; if you cannot find it, import it from `simtk.openmm`."

This is the same as what you did in `nanoverreporter.py`.

File: python-libraries/nanover-openmm/tests_nanover_openmm/test_converter.py [hunk 2]
```diff
@@ -12,8 +12,8 @@
     from openmm.app.element import Element
     from openmm.app.topology import Topology
 except (ImportError, ModuleNotFoundError):
-    from simtk.openmm.app.element import Element
-    from simtk.openmm.app.topology import Topology
+    from openmm.app.element import Element
```
Comment (jbarnoud) [inline python-libraries/nanover-openmm/tests_nanover_openmm/test_converter.py:15]: Like above, you do not need the try/except here.

**Review by jbarnoud — CHANGES_REQUESTED** [review]
Thank you @phuddha for the PR. I did not realise we used the old imports in so many places, I was convinced we already dealt with this and only had a couple of edge cases. It is a good thing you scanned through.

There are a couple of instances where the `simtk` in the import was part of a try/except construct to handle the deprecation. You fixed one of them but others stayed in their try/except. Could you fix them, please?

**Comment by jbarnoud** [comment]
This looks good to go. There are still some occurrences left of `simtk` in various places that we need to get rid of:

```
win_compile.ps1
53:python -c "import simtk"

compile.sh
77:	python -c "import simtk" 2>&1 >/dev/null || {

examples/openmm/OpenMM runner.ipynb
31:    "from simtk import unit\n",

python-libraries/nanover-openmm/tests_nanover_openmm/test_potentials.py
4:from simtk.unit import kilojoule_per_mole, nanometer

python-libraries/nanover-openmm/tests_nanover_openmm/test_imd.py
10:from simtk.unit import nanometer

python-libraries/nanover-openmm/tests_nanover_openmm/simulation_utils.py
14:# Prefixed units in `simtk.unit` are added programmatically and are not
16:from simtk.unit import (

examples/ase/openmm_graphene.ipynb
49:    "import simtk.unit as unit\n",

python-libraries/nanover-openmm/src/nanover/openmm/potentials.py
8:from simtk.unit import Quantity, kilojoule_per_mole, nanometer
23:    >>> from simtk.unit import  kilojoule_per_mole, nanometer

python-libraries/nanover-openmm/src/nanover/openmm/imd.py
60:from simtk import unit

examples/ase/openmm_neuraminidase.ipynb
58:    "import simtk.unit as unit \n",
258:      "Warning: importing 'simtk.openmm' is deprecated.  Import 'openmm' instead.\n"

python-libraries/nanover-ase/tests/openmm_ase/simulation_utils.py
16:# Prefixed units in `simtk.unit` are added programmatically and are not
18:from simtk.unit import (

python-libraries/nanover-ase/tests/openmm_ase/test_calculator.py
4:from simtk.unit import (

python-libraries/nanover-ase/src/nanover/ase/openmm/calculator.py
13:from simtk.unit import angstrom, kilojoules_per_mole, kilojoule_per_mole, amu, Quantity
```

Would you mind fixing them too? Otherwise, please open an issue to track the remaining task and I am happy merging this PR as is.

**Comment by Ragzouken** [comment]
> This looks good to go. There are still some occurrences left of `simtk` in various places that we need to get rid of:
> 
> ```
> win_compile.ps1
> 53:python -c "import simtk"
> 
> compile.sh
> 77:	python -c "import simtk" 2>&1 >/dev/null || {
> 
> examples/openmm/OpenMM runner.ipynb
> 31:    "from simtk import unit\n",
> 
> python-libraries/nanover-openmm/tests_nanover_openmm/test_potentials.py
> 4:from simtk.unit import kilojoule_per_mole, nanometer
> 
> python-libraries/nanover-openmm/tests_nanover_openmm/test_imd.py
> 10:from simtk.unit import nanometer
> 
> python-libraries/nanover-openmm/tests_nanover_openmm/simulation_utils.py
> 14:# Prefixed units in `simtk.unit` are added programmatically and are not
> 16:from simtk.unit import (
> 
> examples/ase/openmm_graphene.ipynb
> 49:    "import simtk.unit as unit\n",
> 
> python-libraries/nanover-openmm/src/nanover/openmm/potentials.py
> 8:from simtk.unit import Quantity, kilojoule_per_mole, nanometer
> 23:    >>> from simtk.unit import  kilojoule_per_mole, nanometer
> 
> python-libraries/nanover-openmm/src/nanover/openmm/imd.py
> 60:from simtk import unit
> 
> examples/ase/openmm_neuraminidase.ipynb
> 58:    "import simtk.unit as unit \n",
> 258:      "Warning: importing 'simtk.openmm' is deprecated.  Import 'openmm' instead.\n"
> 
> python-libraries/nanover-ase/tests/openmm_ase/simulation_utils.py
> 16:# Prefixed units in `simtk.unit` are added programmatically and are not
> 18:from simtk.unit import (
> 
> python-libraries/nanover-ase/tests/openmm_ase/test_calculator.py
> 4:from simtk.unit import (
> 
> python-libraries/nanover-ase/src/nanover/ase/openmm/calculator.py
> 13:from simtk.unit import angstrom, kilojoules_per_mole, kilojoule_per_mole, amu, Quantity
> ```
> 
> Would you mind fixing them too? Otherwise, please open an issue to track the remaining task and I am happy merging this PR as is.

Worth clarifying something that wasn't obvious to me: `simtk.unit` is just an alias for `openmm.unit`, which is why it makes sense to address it as part of this issue. I can't see `simtk` it anywhere in our list of dependencies so I have no idea how it's getting installed in the first place..

**Comment by jbarnoud** [comment]
openmm historically was part of the simtk project. IIRC there is no simtk package, but openmm creates the simtk module. It was confusing people so they decided to get rid of the simtk module and make all accesses go through an openmm module instead.

**Review by Ragzouken — APPROVED** [review]

**Comment by Ragzouken** [comment]
All looks good.

**Pull request merged**